src/components/history/exercises-view.jsx

```diff
@@ -1,6 +1,24 @@
-import { Grid, Space, Tag } from "antd-mobile";
+import { AutoCenter, Grid, Space, Tag } from "antd-mobile";
 import { useState } from "react";
 import { SetBadge } from "./set-badge";
+import { InformationCircleFill } from "antd-mobile-icons";
+import { ThemedPopover } from "../shared/themed-popover";
+
+const getExerciseInformation = (exercises, activeExercise) => {
+	const tonnage = exercises.length
+		? exercises[activeExercise].sets?.reduce((tonnage, set) => {
+				return tonnage + (set.weight || 0) * (set.reps || 1);
+		  }, 0)
+		: 0;
+
+	const reps = exercises.length
+		? exercises[activeExercise].sets?.reduce((summaryReps, set) => {
+				return summaryReps + set.reps;
+		  }, 0)
+		: 0;
+
+	return `Tonnage: ${tonnage} kg for ${reps} reps`;
```

Comment:
It is not Ok when kg is 0. 

add condition if weight 0 use -> `${reps} reps`
else -> `Tonnage: ${tonnage} kg for ${reps} reps`